app/models/concerns/manifest_concern.rb

```diff
@@ -2,8 +2,15 @@
 # A concern to be mixed into the SolrDocument class in order to provide
 # convenient accessors for IIIF manifests embeded in a SolrDocument
 module ManifestConcern
+  VALID_IIIF_CONTENT_TYPES = %w(image manuscript map book).freeze
+
+  # even if a URL is present we need to check the contentMetadata type
+  # to determine whether the manifest will resolve at runtime.
   def manifest
-    fetch('iiif_manifest_url_ssi', nil)
+    manifest_url = fetch('iiif_manifest_url_ssi', nil)
+    return if manifest_url.blank?
+    return unless VALID_IIIF_CONTENT_TYPES.include?(first('content_metadata_type_ssm'))
```

Comment:
I wonder if this line could be extracted to another method if that might be useful elsewhere?

Reply:
Ok, I did an extraction for it to a method called `manifest_available?`